### PR TITLE
Fixing logic for stopping worksapce.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Throw an error if workspace is not in the "STOPPED" state.

Testing with the load test. Logs can be found here https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/load%20tests/job/katka-devfile-test/15/console

### CODE REVIEW

File: tests/e2e/utils/workspace/TestWorkspaceUtil.ts [hunk 1]
```diff
@@ -156,14 +156,20 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
...
-                const wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
+                wsStatus = await this.processRequestHandler.get(stopWorkspaceApiUrl);
                 if (wsStatus.data.status === 'STOPPED') {
+                    stopped = true;
                     break;
                 }
                 await this.driverHelper.wait(TestConstants.TS_SELENIUM_DEFAULT_POLLING);
             }
+            if (!stopped) {
+                let waitTime = TestConstants.TS_SELENIUM_PLUGIN_PRECENCE_ATTEMPTS * TestConstants.TS_SELENIUM_DEFAULT_POLLING;
```
Comment (dmytro-ndp) [inline tests/e2e/utils/workspace/TestWorkspaceUtil.ts:170]: Actually, `waitTime` doesn't take into account getting status time.
You can use [`process.hrtime()`](https://stackoverflow.com/a/14551263/2556329) to measure spent time more accurately.

**Review by dmytro-ndp — APPROVED** [review]
Workspace removal through API [had done as expected](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/basic-MultiUser-Che-check-e2e-tests-against-k8s/1993/console):
```12:01:29   Cleanup
12:01:31     ✓ Remove test workspace (2688ms)
```

File: tests/e2e/utils/workspace/TestWorkspaceUtil.ts (hunk 1 above)
Comment (Katka92) [reply tests/e2e/utils/workspace/TestWorkspaceUtil.ts:170]: I don't think we need that precision info, I wanted to print that more like to show what timeouts were set to the user running those tests - so he/she will know if it was waiting for half a minute, two minutes...

Comment (Katka92) [reply tests/e2e/utils/workspace/TestWorkspaceUtil.ts:170]: So, if you don't mind, I'll leave it as it is. But thank you for rising that :)

Comment (dmytro-ndp) [reply tests/e2e/utils/workspace/TestWorkspaceUtil.ts:170]: I don't mind to leave it as it.

**Pull request merged**